.github/workflows/test_on_push.yml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
     - main
+    - pybammcookie
```

Comment:
I'm not sure why this is required?

Reply:
Oops I was testing it locally forgot to remove it 😅